api/app/db/models/weather_models.py

```diff
@@ -31,6 +31,41 @@ class ProcessedModelRunUrl(Base):
     # Date this record was updated.
     update_date = Column(TZTimeStamp, nullable=False)
 
+class SavedModelRunForSfmsUrl(Base):
```

Comment:
```suggestion
class SavedModelRunForSFMSUrl(Base):
```
Minor nit: Uppercase SFMS